style.css

```diff
@@ -250,3 +236,329 @@ body {
+  grid-template-columns: repeat(auto-fit, minmax(300px, 1fr));
+  gap: 30px;
+}
+
+.card-grid {
+  display: grid;
+  grid-template-areas:
+    "card-img card-img card-img"
+    "card-title card-title card-title"
+    "tags tags tags"
+    "button button button";
+  grid-template-rows: 8px 1fr 7px;
+  grid-template-columns: 20% 1fr 15%;
+  grid-row-gap: 5px;
+  grid-column-gap: 5px;
+  height: 30vh;
+  margin: 0;
+}
+
+/* Card styling */
+.card {
+  position: relative;
+  overflow: hidden;
+  border-radius: 10px;
+  padding: 4px 2px;
+  display: flex;
+  flex-direction: column;
+  margin: 10px;
+  border: 1px solid #ccc;
+}
+
+.card .card-img {
+  grid-column: 1 / 4;
+  background-image: url('./assets/images/img-placeholder.png');
+  background-size: cover;
+  background-position: center;
+  height: 200px;
+  object-fit: cover;
+}
+
+.card-content {
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  width: 100%;
+  padding: 20px;
+  background-color: #fff;
+  transition: transform 0.3s ease-out;
+  align-items: center;
+  text-align: center;
+}
+
+.card-title {
+  color: #3a4a42;
+  width: 100%;
+  height: 57px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.card .overlay h3 {
+  word-wrap: break-word;
+}
+
+/* .tags{ */
+
+.tags {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  margin-top: 10px;
+  border-radius: 4px;
+  margin-bottom: 30px;
+  margin-left: 0;
+}
+
+.tags span.works {
+  background-color: #c6c6c6;
+  border-radius: 6px;
+  padding: 4px 8px;
+  margin-right: 5px;
+  color: black;
+}
+
+.tags span.about {
+  background-color: #fff;
+  border-radius: 6px;
+  padding: 4px 8px;
+  margin-right: 5px;
+  color: #36b37e;
+}
+
+.tags span:first-child {
+  margin-left: 0;
+}
+
+.tags span:last-child {
+  margin-right: 0;
+}
+
+.project-btn {
+  position: relative;
+  background-color: #36b37f;
+  color: #fff;
+  padding: 8px 18px;
+  border-radius: 5px;
+  border: none;
+  cursor: pointer;
+  font-size: 11px;
+  bottom: 8px;
+  width: 122px;
+  height: 48px;
+  align-items: center;
+}
+
+/* about me  */
+
+.about-me .project-btn::after {
+  content: "";
+  position: absolute;
+  top: -10px;
+  left: -150px;
+  width: 50px;
+  height: 50px;
+  background-image: url('./assets/images/about-bg-left.svg');
+  background-size: cover;
+  background-repeat: no-repeat;
+}
+
+.header-content h1:nth-of-type(2) {
+  color: #70c572;
+}
+
+/* about me */
+.aboutme-container {
+  position: relative;
+  width: 100%;
+  max-width: 600px;
+  text-align: center;
+  z-index: 1;
+  align-items: center;
+  margin: auto;
+  padding: 0 18px;
+  margin-bottom: 50px;
+}
+
+.card-img-content {
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  width: 100%;
+  padding: 20px;
+  background-color: #fff;
+  transition: transform 0.3s ease-out;
+  align-items: center;
+  text-align: center;
+}
+
+/* About card */
+.about-card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
```

Comment:
- [ ] Make the cards on the About section responsive until the desktop screen size.
- [ ] Kindly design each card by strictly following every detail(size, font, color, margin, padding) in the original Figma design

> Figma Designs
<img width="297" alt="Screenshot 2023-04-29 at 13 50 58" src="https://user-images.githubusercontent.com/71140133/235303469-afb28d10-ac73-4bf4-a174-5961e694b375.png">

<img width="307" alt="Screenshot 2023-04-29 at 13 50 06" src="https://user-images.githubusercontent.com/71140133/235303447-bce996cb-85e8-4eb6-b721-d3d6501f1dd1.png">

> Your design
<img width="373" alt="Screenshot 2023-04-29 at 13 51 20" src="https://user-images.githubusercontent.com/71140133/235303495-b808d917-7167-4039-ac01-475fa6026ff0.png">

<img width="372" alt="Screenshot 2023-04-29 at 13 50 33" src="https://user-images.githubusercontent.com/71140133/235303501-0edf73bf-665c-4d82-b51e-1339b962fef4.png">